code-challenges.test.js

```diff
@@ -1,7 +1,7 @@
 // ASSESSMENT 2: Coding Practical Questions with Jest
 
-const { workerData } = require("worker_threads")
-const { string } = require("yargs")
+// const { workerData } = require("worker_threads")
```

Comment:
caught the yargs!

Reply:
all I had to do was comment this out lol. I will try and use more higher order functions.